recipes/r-imager/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set version = '0.42.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-imager
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/imager_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/imager/imager_{{ version }}.tar.gz
+  sha256: 6fc308153df8251cef48f1e13978abd5d29ec85046fbe0b27c428801d05ebbf3
+
+build:
+  merge_build_host: True  # [win]
+  skip: True  # [osx]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - xorg-libxfixes  # [linux]
+  host:
+    - r-base
+    - r-rcpp >=0.11.5
+    - r-downloader
+    - r-igraph
+    - r-jpeg
+    - r-magrittr
+    - r-png
+    - r-purrr
+    - r-readbitmap
+    - r-stringr
+    - libtiff
+    - xorg-libx11  # [linux]
```

Comment:
Should this be a cdt? @conda-forge/help-r Anyone know how that should be handled?

Reply:
Looks correct. Also a linkage analysis looks good https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=236787&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=1507